_pages/web-components.md

```diff
@@ -1,53 +0,0 @@
----
```

Comment:
Looks like we still have a link to this file in `frontend.md`

Reply:
Good catch – thanks!